source/dofs/dof_handler.cc

```diff
@@ -2891,16 +2906,16 @@ DoFHandler<dim, spacedim>::pre_distributed_transfer_action()
 
   active_fe_index_transfer->cell_data_transfer = std::make_unique<
     parallel::distributed::
-      CellDataTransfer<dim, spacedim, std::vector<unsigned int>>>(
+      CellDataTransfer<dim, spacedim, std::vector<types::fe_index>>>(
```

Comment:
This means we also need to update the version in the metafile for parallel distributed serialization. I haven't done this just yet, as I wanted to open more PRs after this one.

But we can update the version already in this PR. What do you prefer?